packages/js/src/api/inbox-service.ts

```diff
@@ -143,4 +150,22 @@ export class InboxService {
       actionType,
     });
   }
+
+  fetchPreferences(): Promise<PreferencesResponse[]> {
+    return this.#httpClient.get(`${INBOX_ROUTE}/preferences`);
```

Comment:
Please introduce a buildUrl helper to do this URL concatenation as part of a separate task.

```suggestion
    return this.#httpClient.get(`${INBOX_ROUTE}/preferences`);
```